packages/server/test/unit/LICENSE.md

```diff
@@ -0,0 +1,34 @@
+Unless mentioned below, all tests are covered by the license in the root of this
```

Comment:
Move this bit into the top of the `replace_stream_spec.js` file since that's the only place that uses it. No need to create an external file, so you can remove this `LICENSE.md` file.